guides/common/modules/proc_filter-errata-by-type-severity.adoc

```diff
@@ -0,0 +1,19 @@
+[id="Filter_errata_by_type_or_severity_{context}"]
+= Filter Errata by Type or Severity
+
+Use the following procedure to filter errata by type or severity.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts > All Hosts* and click the name of the host.
+. In the new hosts page click the *Contents* tab.
+. The *Errata* tab of the *Contents* page lists the errata associated with that host.
+. At the top of the page filter by Type by clicking the *Type* button.
+. You can filter to display errata of type Security, Bugfix or Enhancement
+. Filter by severity by clicking the *Severity* button.
+. You can filter to display errata of severity N/A, Low, Moderate, Important or Critical.
+. To unselect your choice, return to the list of options and click the selected option again.
+
+You can use the *Errata* card on the new Host page to pre-filter errata for type before display.
```

Comment:
I don't like the `new Host page` wording. cc @melcorr 

IMHO it should be just "host page", because the docs will only be for Foreman with the "new" host page, aka. its normal and only host page.

Reply:
Yeah there is literally no good reason why this is capitalist. It is just a host page. The UI is new, but that is for the marketing team not the docs team.